docs/pages/database-access/guides/ha.mdx

```diff
@@ -4,14 +4,58 @@ description: How to configure Teleport database access in a Highly Available (HA
 ---
 
 You can deploy the Teleport Database Service in a Highly Available (HA)
-configuration in a couple of common ways: combined replicas and separate
-replicas. Both of those revolve around pointing multiple Database Services to
-the same database instance.
+configuration using two methods, both of which involve pointing multiple
+Database Service instances to the same database instance:
 
-## Combined replicas
+- Deploying multiple Database Service instances with the same dynamic resource
+  watchers 
+- Deploying multiple Database Service instances configured to proxy the same
+  database instances
 
-The most common approach is to assign the same `name` to each Database Service
-proxying the database.
+## Dynamic resource watchers
+
+You can configure multiple Database Service replicas to watch for the same
+dynamic `db` resources. All replicas will detect `db` resources in your cluster
+and proxy connections to the database instances configured via these resources.
+
+To configure the Database Service to watch for dynamic resources, add a `labels`
+field to its `resources` configuration, for example:
+
+```yaml
+db_service:
+  enabled: "yes"
+  resources:
+  - labels:
+      "*": "*"
+```
+
+When you create a dynamic `db` resource, all Database Service replicas
+configured to watch that `db` resource's labels will proxy the corresponding
+database instance.
+
+When connecting, Teleport will randomly pick the Database Service instance to
+connect through to provide some load balancing. If the selected instance is down
+(e.g. in case of AZ outage), Teleport will try to connect via other instances.
```

Comment:
What's an AZ outage?

Reply:
Availability zone. I'll use "data center" to be more general.